app/src/main/java/fr/free/nrw/commons/review/ReviewImageFragment.java

```diff
@@ -40,6 +42,9 @@
     @BindView(R.id.button_no)
     Button noButton;
 
+    private final String SAVED_USER = "saved_user";
+    private String user;
```

Comment:
Javadoc. Please make sure the difference with the `user` variable is easy to understand for any newcomer.